src/Common/Auth/OpenIDConnect/Repositories/ClientRepository.php

```diff
@@ -12,6 +12,7 @@
 
 namespace OpenEMR\Common\Auth\OpenIDConnect\Repositories;
 
+use http\Exception\RuntimeException;
```

Comment:
@adunsulag , Just confirming that above line is a real thing. Tried to look it up on google and couldn't find it.

Reply:
@bradymiller Shoot, wrong class.  Should be this one: https://www.php.net/manual/en/class.runtimeexception.php  

```use \RuntimeException```

Not sure why phpstorm auto inserted that one.  Good catch.